ProyectoSemestral3/Assets/Scripts/PlayerMovement.cs

```diff
@@ -15,7 +15,7 @@ private void Start()
     {
         controller = gameObject.AddComponent<CharacterController>();
```

Comment:
All class parameters should have this format `_ controller  = null`, speed, isItGrounded, etc the same.

Reply:
Also change `public` to `[SerializeField] private`

Reply:
Hi teacher, I changed everything on the movement and camera, I removed the run capability and also removed the jump so that I can look forward on investigating more about those mecanics